metadata-ingestion/tests/integration/kafka-connect/setup/conf/mongo-init.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+mongo -- "$MONGO_INITDB_DATABASE" <<-EOJS
+    conn = new Mongo();
+    db = conn.getDB("test_db");
+    db.purchases.insertOne({ _id: 3, item: "lamp post", price: 12 });
+    db.purchases.insertOne({ _id: 4, item: "lamp post", price: 13 });
+EOJS
+
+
+{
```

Comment:
Hey @frsann I noticed that `rs.initiate()` is rather called after mongo db container has already started (initialized). I would suggest - calling the init script using docker exec from test file - refer [this](https://github.com/datahub-project/datahub/blob/29c6c43cb319shall be644323c61999c4daaf95335d93f0/metadata-ingestion/tests/integration/sql_server/test_sql_server.py#L25-L27) for example to avoid ocaasional failures of the test.